test/knex.test.js

```diff
@@ -259,6 +260,77 @@ describe('transaction', function () {
 
     throw new Error('expected the call to throw')
   })
+
+
+  it('adopt-commit', async () => {
+
+    let foo1_id
+
+    const trx = await knex(DbConfigPG).transaction()
+    const txseneca  = await si.entity.adopt({handle:trx})
+
+    //I'm using the same Knex instance....
+    const foosSave = await txseneca.entity('foo').data$({p1:'t1'}).save$()
+    foo1_id = foosSave.id
+    await txseneca.entity('foo').data$({p1:'t2'}).save$()
```

Comment:
capture the result of this operation:
```
let foo2 = await ....
```

using foo2 since p1=t2

Reply:
Hmm, ok